Corzbank.Data/Entities/Card.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Corzbank.Data.Enums;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace Corzbank.Data.Entities
+{
+    public class Card: KeyEntity
+    {
+        [Required]
+        [StringLength(16, ErrorMessage = "Card length should be 16")]
+        public string CardNumber { get; set; }
+
+        [Required]
+        [StringLength(4, ErrorMessage = "ExpirationDate length should be 4")]
+        public string ExpirationDate { get; set; }
+
+        [Required]
+        [StringLength(3, ErrorMessage = "CvvCode length should be 3")]
+        public string CvvCode { get; set; }
```

Comment:
string?

Reply:
there will be combination of 3 digits

Reply:
If it is a combination of 3 digits, why didn't u marked as int? It's not logical to have only digits inside string
[numeric types](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types) ushort have a range from 0 to 65,535, that's what u need